src/Symfony/Component/HttpKernel/Bundle/Bundle.php

```diff
@@ -153,49 +152,12 @@ public function getParent()
     }
 
     /**
-     * Finds and registers Commands.
-     *
-     * Override this method if your bundle commands do not follow the conventions:
-     *
-     * * Commands are in the 'Command' sub-directory
-     * * Commands extend Symfony\Component\Console\Command\Command
+     * Registers console commands.
      *
      * @param Application $application An Application instance
      */
     public function registerCommands(Application $application)
```

Comment:
This is not good if you don't change the min version to forbid using TwigBundle 4.0 with HttpKernel 3.4 and the no-op was important though